src/plots/cartesian/axes.js

```diff
@@ -2305,16 +2327,37 @@ axes.drawOne = function(gd, ax, opts) {
 
     // this happens when updating matched group with 'missing' axes
     if(!mainPlotinfo) return;
+    // Will this axis 'push' out other axes?
+    ax._shiftPusher = overlayingShiftedAx.includes(ax._id) || overlayingShiftedAx.includes(ax.overlaying) || ax.shift === true;
```

Comment:
`includes` is not supported on IE. Also we could test `ax.shift` first to be faster.
```suggestion
    ax._shiftPusher = ax.shift === true || 
        overlayingShiftedAx.indexOf(ax._id) !== -1 || 
        overlayingShiftedAx.indexOf(ax.overlaying) !== -1;
```